source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst

```diff
@@ -710,6 +710,16 @@ Why is waitQueueTimeoutMS optional for some drivers?
 
 We are anticipating eventually introducing a single client-side timeout mechanism, making us hesitant to introduce another granular timeout control. Therefore, if a driver/language already has an idiomatic way to implement their timeouts, they should leverage that mechanism over implementing waitQueueTimeoutMS.
 
+Why is there a distinction between a Pool being "closed" and the Pool resources being free-d / garbage collected? (a.k.a. why does a Pool have a "half-closed" state?)
```

Comment:
Nit: the phrase half-closed doesn't appear anywhere in the spec.  Should it?  Or is there some other phrase or section that we can refer to here that clarifies what part of the spec this is explaining?